minidump-common/src/traits.rs

```diff
@@ -79,6 +79,12 @@ where
         input.sort_by_key(|x| x.0);
         let mut vec: Vec<(Range<u64>, V)> = Vec::with_capacity(input.len());
         for (range, val) in input.into_iter() {
+            if range.is_none() {
+                warn!("Unable to create valid range for {:?}", val);
+                continue;
+            }
+            let range = range.unwrap();
```

Comment:
```suggestion
            let range = match range {
                Some(range) => range,
                None => {
                warn!("Unable to create valid range for {:?}", val);
                continue;
                }
            };
```

maybe instead of the unwrap

Reply:
I go back and forth on this kinda thing a lot, but ultimately I think the unwrap is easier to read. Maybe one day we'll have [let-else](https://github.com/rust-lang/rfcs/pull/3137) to properly express this pattern.